Duplicati/Server/Runner.cs

```diff
@@ -438,7 +439,7 @@ public static Duplicati.Library.Interface.IBasicResults Run(IRunnerData data, bo
 
 
             var backup = data.Backup;
-            Duplicati.Library.Utility.TempFolder tempfolder = null;
+            Duplicati.Library.Utility.TempFolder tempfolder = new Duplicati.Library.Utility.TempFolder();

```

Comment:
This will cause it to create a temporary folder, even when it is not needed. The temp folder is only required when including the job setups inside the backup.